Release.toml

```diff
@@ -142,3 +142,5 @@ version = "1.9.1"
 ]
 "(1.9.0, 1.9.1)" = [
 ]
+"(1.9.1, 1.10.0)" = [
```

Comment:
Nit: Could we put the square brackets on a single line to stay consistent with previous empty migration lists? :)